### PR TITLE
Removes shuttle templates which are unbalanced and shitty

### PR DESCRIPTION
**Pull request by Niknakflak**
The supermatter shuttle was removed because although it's supposed to be a meme.  It's an incredibly lethal meme and falls under that "should never be touched by an admin" thing.  It's shitty and needs to be taken out.

The clown shuttle is horribly mapped.  It makes the shuttle brig useless, the shuttle bridge doesn't even have access restrictions and it's a shitty meme and it's annoying.  Chasms in a shuttle brig and unrestricted bridge is shit and it should be toasted.

The room shuttle doesn't even have a shuttle brig, bridge, or enough seats for the crew.  It's shitty, memey, and has a shitty inside joke inside written on a piece of paper in all lowercase.

Golden merge token located here: https://www.tgstation13.org/phpBB/viewtopic.php?f=55&t=6773&start=150#p182620  bestowed upon me by headcoder himself

### CODE REVIEW

File: code/datums/shuttles.dm [hunk 1]
```diff
@@ -105,33 +91,9 @@
...
-		\"Are you sure?\"\n\
-		\"Yes, it has a safety record of N-A-N, which is apparently \
-		larger than 100%.\""
-	admin_notes = "Supermatter that spawns on shuttle is special anchored \
-		'hugbox' supermatter that cannot take damage and does not take in \
-		or emit gas. Outside of admin intervention, it cannot explode. \
-		It does, however, still dust anything on contact, emits high levels \
-		of radiation, and induce hallucinations in anyone looking at it \
-		without protective goggles. Emitters spawn powered on, expect \
+s spawn powered on, expect \
```
Comment (Niknakflak) [inline code/datums/shuttles.dm]: Is this a mistake

**Comment by lzimann** [comment]
:salt:

**Comment by Cheridan** [comment]
i should have known my golden token would only be used for evil

**Comment by coiax** [comment]
> It's an incredibly lethal meme

> It makes the shuttle brig useless, the shuttle bridge doesn't even have access restrictions

image

Also, for once, I will defend goof's Space Apartment as brilliant design. I have seen the population of a medium sized round cram into that and devolve almost instantly into a brawl that lasted until Centcom.

**Comment by xxalpha** [comment]
This kind of fun removal really shouldn't be allowed, meme tokens or no.

**Comment by lzimann** [comment]
where's the i ded tag

**Comment by Niknakflak** [comment]
You think it's fun but no one enjoys dying the second they got on the shuttle and no one enjoys their round being ruined because an admin thought they would be funny.  

shuttles aren't the problem
Bad shuttles are the problem

**Comment by xxalpha** [comment]
How is the entire round ruined if at the very end you can't escape alive?

I assume you mean traitors will get upset. How about admins don't use the dangerous shuttles in traitor rounds and such?

**Comment by Jalleo** [comment]
Maybe we should make each time a emergency shuttle is called be a different template and remove admins from the decision process.

**Comment by Niknakflak** [comment]
Admins see stupid thing, admin does stupid thing
These shuttles are bad anyway, don't even try that

**Comment by Hatfish** [comment]
if you're that worried about escaping alive then why don't you just take an escape pod?

**Comment by xxalpha** [comment]
>These shuttles are bad anyway

That's just like, your opinion.

**Comment by Niknakflak** [comment]
>forcing players to change just because of memes
>people value memes over immersions and allowing players to take the escape shuttle

This is what we have come to

**Comment by xxalpha** [comment]
Nobody is forcing you to change the shuttles. If you can't help yourself then get deadminned.

**Comment by lzimann** [comment]
he is complaining because another admin changed the shuttle and he died

**Comment by ChangelingRain** [comment]
Really, I just need to point at the tag.

**Comment by Niknakflak** [comment]
reopen this

**Comment by Cheridan** [comment]
don't reopen this niknak is spamming me

**Comment by KorPhaeron** [comment]
I don't see the point of having shuttles that just kill everyone in the codebase

**Comment by Dorsisdwarf** [comment]
Yeah but this removes Roomshuttle which has apparently made one round fun so far

**Comment by Niknakflak** [comment]
This removes a shitty meme that doesn't have a shuttle brig

**Comment by Dorsisdwarf** [comment]
Not all admin-spawned shuttles must be fully functional

**Comment by Niknakflak** [comment]
There's really no point to it

**Pull request closed without merging**